content/advanced/330_servicemesh_using_appmesh/canary_deployment/testing_v2.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Testing Canary"
+date: 2020-01-27T08:30:11-07:00
+weight: 100
+---
+
+Get the Loadbalancer endpoint
+```bash
+export LB_NAME=$(kubectl get svc ingress-gw -n prodcatalog-ns -o jsonpath="{.status.loadBalancer.ingress[*].hostname}") 
+echo $LB_NAME
+```
+
+Go to the above Load Balancer endpoint URL, you should see below screen which shows that Catalog Detail `proddetail-v1` Version 1 is being accessed.
+
+![frontend](/images/app_mesh_fargate/lbfrontend.png)
+
+Now click on the button **Canary Deployment** (may be couple of times), you should see Product Catalog `proddetail-v2` Version 2 is being accessed. You can see the `XYZ.com` vendor in the detail for version 2.
+
+![canaey](/images/app_mesh_fargate/canary.png)
+
+If anything were to go wrong, you can simply rollback to the known-good v1 version of the services. Once you've verified things are good with the new versions, you can shift all traffic to them and deprecate v1.
```

Comment:
Here could be a good spot to talk about monitoring health, increase from 10->20% etc... but, it's fine as it is as well for now.